sdk/storage/Azure.Storage.Files.DataLake/tests/FileClientTests.cs

```diff
@@ -254,6 +254,7 @@ public async Task Ctor_DefaultAudience()
         }
 
         [RecordedTest]
+        [PlaybackOnly("DefaultAzureCredential does not play nicely with this test.")]
```

Comment:
Should we create a github issue to eventually reenable this test?

Reply:
https://github.com/Azure/azure-sdk-for-net/issues/44967, and updated the `PlaybackOnly` annotations to refer to this issue.